tests/syrupy/extensions/amber/__snapshots__/test_amber_filters.ambr

```diff
@@ -1,36 +1,36 @@
 # name: test_filters_error_prop[path_filter]
-  <class 'WithNested'> {
+WithNested(
```

Comment:
(note) Okay I can see how this is nicer 😄 

What happens if you have a multiple line string in the attribute though?

Reply:
You can see an example in one of the test cases, where sets and tuples were used as keys [`test_amber_serializer::test_dict[actual2]`](https://github.com/tophat/syrupy/blob/fd878878d107d6fb5b57004e89452013f2f52935/tests/syrupy/extensions/amber/__snapshots__/test_amber_serializer.ambr#L137). Will add a multiline string case too.